Card/src/main/java/com/paypal/android/card/Card.kt

```diff
@@ -0,0 +1,7 @@
+package com.paypal.android.card
+
+data class Card(
```

Comment:
We may have some conflicts with the current Card PR. But they should be simple to resolve.

Reply:
Yeah we only have `cardNumber` and `expirationDate` at the moment.